frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -653,24 +653,28 @@ pub mod pallet {
 		}
 
 		fn offchain_worker(now: T::BlockNumber) {
+			let mut acquired = false;
 			match Self::current_phase() {
 				Phase::Unsigned((true, opened)) if opened == now => {
 					// mine a new solution, cache it, and attempt to submit it
-					let initial_output = Self::try_acquire_offchain_lock(now)
-						.and_then(|_| Self::mine_check_save_submit());
-					log!(info, "initial OCW output at {:?}: {:?}", now, initial_output);
+					let initial_output = Self::try_acquire_offchain_lock(now).and_then(|_| {
+						acquired = true;
+						Self::mine_check_save_submit()
```

Comment:
mine_check_save_submit is executing `unsigned_pre_dispatch_checks` through basic checks as far as I can see